src/main/java/kotlinx/reflect/lite/impl/KotlinReflectionInternalError.kt

```diff
@@ -9,4 +9,4 @@ package kotlinx.reflect.lite.impl
  * Signals that Kotlin reflection had reached an inconsistent state from which it cannot recover.
  * @suppress
  */
-class KotlinReflectionInternalError(message: String) : Error(message)
+public class KotlinReflectionInternalError(message: String) : Error(message)
```

Comment:
Not sure about this one, but I guess errors that the user can end up seeing should be public, even if there's nothing sensible they can do about it?

Reply:
The honest question is "we don't know yet".

>I guess errors that the user can end up seeing should be public

Not necessary, as long as it has a meaningful (for the API) superclass/superinterface.
This one is weird -- it takes its original place in `kotlin.reflect.jvm.internal` package and probably was made `public` by mistake, but people are [already using it](https://grep.app/search?current=2&q=KotlinReflectionInternalError&filter[lang][0]=Kotlin)